src/Build.UnitTests/Graph/GetCompatiblePlatformGraph_Tests.cs

```diff
@@ -51,7 +55,6 @@ public void ResolvesViaPlatformLookupTable()
                                                     </Project>");
 
                 IDictionary<string, string> globals = new Dictionary<string, string>();
```

Comment:
low pri nit to only do if you're going to make another commit anyway: no need for this var

Reply:
removed. going to do a squash once i have all the changes made